vagrant_support/cron.yml

```diff
@@ -1,44 +1,51 @@
 - name: set cron mailto
   cron:
+    name: set cron mailto
```

Comment:
It bugs me that they can't inherit the name from the parent, but I guess that's how ansible is.